.circleci/config.yml

```diff
@@ -1,46 +1,111 @@
 version: 2.1
 
-var_1: &working_dir ~/src
-var_2: &dapp_working_dir ~/src/raiden-dapp
+parameters:
+  base_executor_image_tag:
+    type: string
+    default: lts-browsers
+
+  e2e_environment_executor_image_tag:
+    type: string
+    default: latest
+
+  working_directory:
+    type: string
+    default: ~/src
+
+  yarn_cache_key:
+    type: string
+    default: raiden-{{ .Branch }}-{{ checksum "yarn-lock.yaml" }}
```

Comment:
As discussed here the `Branch` doesn't make much sense: https://github.com/raiden-network/light-client/pull/2312#discussion_r516599517

Reply:
Good catch. Sorry that I missed that while resolving the conflicts. :see_no_evil: 

It is a very good feeling that the reviewer is really competent and careful. Thanks for being such a colleague! :pray:

Reply:
While looking again on the master version: why do we have two keys here, while [the second](https://github.com/raiden-network/light-client/blob/5a729c9e3b1ce6fd1ff1ad71771a66811951d0b5/.circleci/config.yml#L77) looks pretty awkward. Especially because we don't use this key a few lines below. Do you have any insights?

Reply:
I assumed this was kind of a fallback, but now that you mention it I'm not so sure any more.

Reply:
I asked Andre as he is the author and I got [this helpful link](https://circleci.com/docs/2.0/caching/#restoring-cache) to the documentation. So I'll recover that as well. :+1:

Reply:
:heavy_check_mark: